connectivity_summary.py

```diff
@@ -352,11 +352,18 @@ def path(self):
         if self._site_path is None:
             date, slice, site = self.expt_id.split('-')
             root = os.path.dirname(self.entry.file)
-            paths = [
-                os.path.join(root, date+"_000", "slice_%03d"%int(slice), "site_%03d"%int(site)),
-                os.path.join(root, 'V1', date+"_000", "slice_%03d"%int(slice), "site_%03d"%int(site)),
-                os.path.join(root, 'ALM', date+"_000", "slice_%03d"%int(slice), "site_%03d"%int(site)),
-            ]
```

Comment:
A good guideline for coding is "do not repeat yourself". Can you simplify this and avoid copying the `path =` definition?